code/michael/html_css_flask/html_blog_lab_2/templates/blog.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" type="text/css" href="{{ url_for('static', filename='style.css') }}"/>
+</head>
+<body>
+    <header>
+        <div id="top">
+
+        <a href="blog.html"><img class="logo" src="../static/logo.png" alt="Logo"></a>
+        <p class="slogan"><em>Blogging is Life</em></p>
+        </div>
+        <div class="topnav">
+
+        </div>
+        <h1 class="title">BLOG</h1>
+
+    </header>
+    <div class="mid">
```

Comment:
Missing semantic element <section> You can actually replace this <div> on line 23 and it won't change your code but you'll meet the section requirement. Sections can be used to separate content.